src/EnergyPlus/Boilers.cc

```diff
@@ -227,6 +227,12 @@ void GetBoilerInput(EnergyPlusData &state)
             ShowContinueError(state, format("Invalid {}={:.3R}", state.dataIPShortCut->cNumericFieldNames(2), state.dataIPShortCut->rNumericArgs(2)));
             ShowSevereError(state, format("...{} must be greater than 0.0", state.dataIPShortCut->cNumericFieldNames(2)));
             ErrorsFound = true;
+        } else if (state.dataIPShortCut->rNumericArgs(2) > 1.0) {
+            ShowWarningError(state,
+                             fmt::format("{} = {}: {} should not typically be greater than 1.",
```

Comment:
For these warnings it would be helpful to include the actual input value.
```
   ** Warning ** WaterHeater:Mixed = MIXED GAS TANK: Heater Thermal Efficiency=1.1 should not typically be greater than 1.
```